utils.sh

```diff
@@ -72,6 +72,11 @@ function create_cluster() {
       mv ${assets_dir}/worker.ign ${assets_dir}/worker.ign.orig
       jq -s '.[0] * .[1]' ${IGNITION_EXTRA} ${assets_dir}/worker.ign.orig | tee ${assets_dir}/worker.ign
     fi
+
+    # Preserve the assets for debugging
+    mkdir -p "${assets_dir}/saved-assets"
+    cp -av "${assets_dir}/openshift/" "${assets_dir}/saved-assets"
```

Comment:
There is also the `${assets_dir}/manifests` directory which I think also gets removed after `create cluster` ?

Reply:
Added